profile/README.md

```diff
@@ -2,12 +2,25 @@
 <img alt="Logo Banner" src="https://raw.githubusercontent.com/jellyfin/jellyfin-ux/master/branding/SVG/banner-logo-solid.svg?sanitize=true"/>
 </p>
 
-[Jellyfin is a Free Software Media System](https://jellyfin.org) that puts you in control of managing and streaming your media. Provide streaming media
-from a dedicated server to end-user devices via multiple apps. Jellyfin is build on the .NET Core framework to enable
-full cross-platform support. There are no strings attached, no premium licenses or features, and no hidden agendas:
-just a team who want to build something better and work together to achieve it. We welcome anyone who is interested in
-joining us in our quest!
-
-For further details, please see [our documentation page](https://jellyfin.org/docs/). To receive the latest updates,
-get help with Jellyfin, and join the community, please visit
-[one of our communication channels](https://jellyfin.org/contact).
+[Jellyfin is the Free Software Media System](https://jellyfin.org) that puts you in control of managing and streaming your media.
+
+Jellyfin provides media streaming from a dedicated, self-hosted server to end-user devices via multiple client apps or the web.
+The [Jellyfin server](https://github.com/jellyfin/jellyfin) is built on the .NET Core framework to enable full cross-platform
+operation, and supports multiple flavours of Linux, Windows, and MacOS as a server. Plugins, both 1st and 3rd party, provide
+additional functionality for everything from reporting, to additional metadata, to authentication. We provide clients for many
+common devices including [Web browsers](https://github.com/jellyfin/jellyfin-web), [Desktops](https://github.com/jellyfin/jellyfin-media-player),
```

Comment:
```suggestion
common devices including [Web browsers](https://github.com/jellyfin/jellyfin-web), [Desktop](https://github.com/jellyfin/jellyfin-media-player) (Windows, Linux, macOS),
```

Or instead of mentioning the individual platforms (Cross-Platform)